source/js/components/currency-selector.js

```diff
@@ -205,7 +205,13 @@ class CurrencySelect {
     presets.forEach(input => {
       input.addEventListener("change", _ => {
         if (input.checked) {
-          otherAmountInputs.forEach(other => (other.value = ""));
+          otherAmountInputs.forEach(other => {
+            other.value = "";
+            var radio = other.parentNode.querySelector(
```

Comment:
let's move this up to outside of the `presets.forEach` on line 205, because it's a fixed HTML element, so we don't need to query for it on every iteration. Also, let's call it something indicative of its role, like `customAmount`.

Reply:
fixed.